### ISSUE TITLE
update("$push") does not work with Persistable

### ISSUE THREAD

**Issue by minenok-tutu** [mongodb/mongo-php-driver#207]
I have document structure similar to that:
```class A {
    /** @var B[] */
    private $_b = [];
}```
Both A and B are persistable. So, when I insert A with multiple Bs via driver, it's OK. I see __pclass field in mongo shell.
When I execute something like (of course, again via php api)
`update({"$push": {"b": {new B object}}})`
no __pclass field is created and unserialization goes wrong.

**Comment by gquemener** [comment]
Might be related to https://jira.mongodb.org/browse/PHPC-550

**Comment by jmikola** [comment]
This should be resolved by [PHPC-550](https://jira.mongodb.org/browse/PHPC-550), which will be included in the forthcoming 1.1.3 release this week.

**Comment by jmikola** [comment]
[1.1.3](https://github.com/mongodb/mongo-php-driver/releases/tag/1.1.3) is now released.
